### ISSUE TITLE
Clearer Readme

### ISSUE THREAD

**Issue by sayan1999** [fossfreedom/smallwindow#6]
I cloned the repo to '$HOME/.local/share/rhythmbox/plugins' along with other plugins. and activated the plugin from rhythmbox console. I don't see any small window appearing. 

Expectations could be mentioned in the readme with a screenshot. Also I seek a bit guide on how to have the small window.

**Comment by gamesbook** [comment]
Agreed. I activated the plugin... now what?  Cannot see how to switch to a "Small Window" mode?

**Edit:** OK, I found it under "Tools" and it works (Rhythmbox Version 3.4.4).  But it would be great if remembered its last position and defaulted to that instead of popping back to the centre.
